Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs

```diff
@@ -560,6 +561,19 @@ public void TryUseInteraction(IEntity user, IEntity used)
         /// </summary>
         public void UseInteraction(IEntity user, IEntity used)
         {
+
+            if (used.TryGetComponent<UseDelayComponent>(out var delayComponent))
+            {
+                if (delayComponent._activeDelay)
```

Comment:
It would probably be best if there was a convenience method to use here so you didn't have to write out all these lines of code every time.